### PR TITLE
importUsage imported from vleet

### PR DESCRIPTION
**Pull request by rjacolin**

### CODE REVIEW

File: lib/airvantage.js [hunk 1]
```diff
@@ -194,6 +194,27 @@ _.each(SYSTEM_LIFECYCLE_OPERATIONS, function(operationName) {
...
+            query: this._buildQuery(),
+            headers: _.extend(this._buildHeaders(), {
+                "Content-Type": "application/octet-stream"
+            }),
+            json: true,
+            body: data
+        })
+        .then(function(res) {
+            return res.body;
+        }).catch(error => {
```
Comment (sebz) [inline lib/airvantage.js]: This is a new syntax of node 4.x. Please use the old one for the moment:

```js
.catch(function(error) {
   console.error(error.response.body);
})
```

**Comment by sebz** [comment]
@rjacolin I'll merge this once you fix the `catch` clause.

**Comment by rjacolin** [comment]
Done

**Comment by sebz** [comment]
LGTM :+1:

**Pull request merged**